### ISSUE TITLE
Install instructions don't work

### ISSUE THREAD

**Issue by regentcid434** [smebdi/react-native-signature-capture#1]
Following the included npm instructions installs a different package with the same name. Please rectify.

**Comment by calebmackdavenport** [comment]
@regentcid434 please follow expected new issue template, _thanks_. 

## Expected Behavior


## Current Behavior


## Possible Solution


## Steps to Reproduce


1.
2.
3.
4.

## Context (Environment)





## Detailed Description


## Possible Implementation

**Comment by regentcid434** [comment]
@calebmackdavenport Still waiting on an adequate response from the dev team.
